src/main/java/com/formulasearchengine/mathmltools/mml/CMMLInfo.java

```diff
@@ -139,15 +101,11 @@ private void abstractNodeCD(Node node) {
         String cd;
         try {
             cd = node.getAttributes().getNamedItem("cd").getNodeValue();
-        } catch (final DOMException e) {
+        } catch (final DOMException | NullPointerException e) {
             //TODO: Implement CD fallback
```

Comment:
is that really a program error? I guess it might be triggered by a data error. So I would just use a warning or a lower log level